docs/SIL.rst

```diff
@@ -4372,6 +4373,10 @@ a field declaration -- which references the desired sub-field in source variable
 In the snippet above, source variable "the_struct" has an aggregate type ``$MyStruct`` and we use a SIL DIExpression with ``op_fragment`` operator to associate ``%1`` to the ``y`` member variable (via the ``#MyStruct.y`` directive) inside "the_struct".
 Note that the extra source location directive follows right after ``name "the_struct"`` indicate that "the_struct" was originally declared in line 8, but not until line 9 -- the current ``debug_value`` instruction's source location -- does member ``y`` got updated with SSA value ``%1``.
 
+For tuples, it works similarly, except we use ``op_tuple_fragment``, which takes two arguments: the tuple type and the index. If our struct was instead a tuple, we would have:
+
+  debug_value %1 : $Int, var, (name "the_tuple", loc "file.swift":8:7), type $(x: Int, y: Int), expr op_tuple_fragment:$(x: Int, y: Int):1, loc "file.swift":9:4
+
```

Comment:
This is a neat addition to the syntax, but is there a grammatical reason to not just use the same syntax we use for structs?
```
debug_value %1 : $Int, var, (name "the_tuple", loc "file.swift":8:7), type $(x: Int, y: Int), expr op_fragment:#$(x: Int, y: Int).1, loc "file.swift":9:4
```

Reply:
Grammatically no, it's just that the # and dot syntax in SIL are for declaration references, and there is no declaration to refer to here

Reply:
@jckarter Do you have any concerns about this syntactic addition to textual SIL?